Tetris Clone C++/GridController.cpp

```diff
@@ -160,13 +156,13 @@ void GridController::printGrid() {
 
 void GridController::saveGridToFile() {
     FILE *fp = fopen(fileName, "wb");
-    int length = ROWS * COLUMNS;
+    size_t length = ROWS * COLUMNS;
     char *gridString = (char *)malloc(sizeof(char) * length);
     for (int i = 0; i < length; i++) {
         gridString[i] = GridController::characterForType(this->grid[0][i]);
```

Comment:
here and maybe one other place needs to have the loop variable changed to size_t